exla/c_src/exla/custom_calls.h

```diff
@@ -1,9 +1,9 @@
-#pragma once
-
-template <typename DataType>
-void qr_cpu_custom_call(void *out[], const void *in[]);
+#ifndef EXLA_MLIR_CUSTOM_CALLS_H_
+#define EXLA_MLIR_CUSTOM_CALLS_H_
```

Comment:
why the change here?

Reply:
Just for consistency with the other files, I can change any way around if you have a preference :)